src/full_node/mempool_check_conditions.py

```diff
@@ -81,6 +69,18 @@ def mempool_assert_time_exceeds(condition: ConditionVarPair):
     return None
 
 
+def mempool_assert_announcement_consumed(condition: ConditionVarPair, spend_bundle: SpendBundle) -> Optional[Err]:
```

Comment:
Again... you can move this up to make the diff nicer.

Reply:
This code seems really similar to the code above. Why is it separate? (Probable answer: just because all the others are. So maybe my question is why that happened.)